src/InterventionBackend.php

```diff
@@ -267,25 +213,18 @@ public function getImageResource()
             $bytesWritten = file_put_contents($path ?? '', $stream);
             // if we fail to write, then load from stream
             if ($bytesWritten === false) {
-                $resource = $this->getImageManager()->make($stream);
+                $resource = $this->getImageManager()->read($stream);
             } else {
                 $this->setTempPath($path);
-                $resource = $this->getImageManager()->make($path);
-            }
-
-            // Fix image orientation
-            try {
-                $resource->orientate();
-            } catch (NotSupportedException $e) {
-                // noop - we can't orientate, don't worry about it
```

Comment:
`orientate()` was replaced with `orient()` - but that [happens automatically by default](https://image.intervention.io/v3/modifying/effects#image-orientation-according-to-exif-data) now, so we don't have to do it manually here.